types/botvs/index.d.ts

```diff
@@ -392,59 +377,43 @@ declare global {
              * 支持现货(火币/BitVC/OKCoin/OKCoin国际/OKCoin期货/BTCChina/BitYes)市价单, 市价单价格指定为-1
              * exchange.Buy(1000), 指买市价1000元的币, BTCChina例外exchange.Buy(0.3)指市价买0.3个币
              *
-             * @param {number} Price
-             * @param {number} Amount
-             * @param {*[]} args
-             * @return {string} - 订单ID
+             * @return - 订单ID
```

Comment:
I have refactor this some minutes before in #20932, but I have not take care about the comments! Should I merge this PR into mine and solve the conflicts?

Reply:
:+1: